src/buildercore/trop.py

```diff
@@ -132,6 +132,16 @@ def rds_security(context):
                           ingress_ports,
                           "RDS DB security group")
 
+def docdb_security(context):
+    """returns a security group for the docdb instance.
+    this security group only allows access within the subnet, not because of the ip address range but because this is dealt with in the subnet configuration"""
+    ingress_data = [27017]
+    ingress_ports = _convert_ports_to_dictionary(ingress_data)
+    return security_group("VPCSecurityGroup",
```

Comment:
I think this name doesn't need to be unique, so it doesn't clash with the RDS security group

Reply:
(it does need to be unique within the same project, but I don't think projects usually contain both RDS and DocumentDB at the same time)

Reply:
Ah, yes you are correct. I've updated it to a more meaning full name.